dictionaries/companies/src/companies.txt

```diff
@@ -708,6 +709,7 @@ Health Net, Inc.
 Healthpeak Properties
 HeBei Iron & Steel Group
 Heineken Holding
+HelloSign
```

Comment:
This is a product and as you suggested, it might be better in the tools dictionary.
In the future, it might be worth it to have product dictionaries based upon industry.